src/EFCore/Query/QueryableMethodTranslatingExpressionVisitor.cs

```diff
@@ -546,128 +587,389 @@ protected override Expression VisitExtension(Expression node)
             return Expression.Field(targetExpression, fieldInfo);
         }
 
+        /// <summary>
+        ///     Translates given subquery by creating a subquery visitor.
+        /// </summary>
+        /// <param name="expression"> The subquery expression to translate. </param>
+        /// <returns> The translation of given subquery. </returns>
         public virtual ShapedQueryExpression TranslateSubquery([NotNull] Expression expression)
         {
             Check.NotNull(expression, nameof(expression));
 
             return (ShapedQueryExpression)CreateSubqueryVisitor().Visit(expression);
         }
 
+        /// <summary>
+        ///     Creates a visitor customized to translate subquery.
+        /// </summary>
+        /// <returns> A visitor to translate subquery. </returns>
         protected abstract QueryableMethodTranslatingExpressionVisitor CreateSubqueryVisitor();
 
+        /// <summary>
+        ///     Creates a <see cref="ShapedQueryExpression"/> for given type by finding it's entity type in the model.
+        /// </summary>
+        /// <param name="elementType"> The clr type of the entity type to look for. </param>
+        /// <returns> A shaped query expression for given clr type. </returns>
         [Obsolete("Use overload which takes IEntityType.")]
         protected abstract ShapedQueryExpression CreateShapedQueryExpression([NotNull] Type elementType);
+        /// <summary>
+        ///     Creates a <see cref="ShapedQueryExpression"/> for given entity type.
```

Comment:
```suggestion
        ///     Creates a <see cref="ShapedQueryExpression"/> for the given entity type.
```

And apply this to all similar docs